src/seals/diagnostics/risky_path.py

```diff
@@ -0,0 +1,58 @@
+"""Environment testing for correct behavior under stochasticity."""
+
+import gym
+import numpy as np
+
+from seals.diagnostics.base_env import BaseEnv
+
+
+class RiskyPathEnv(BaseEnv):
+    """Environment with two paths to a goal: one safe and one risky.
+
+    Many LfH algorithms are derived from Maximum Entropy Inverse Reinforcement
+    Learning (Ziebart et. al, 2008), which models the demonstrator as producing
```

Comment:
Where are (Ziebart, 2008) and (ZIebart, 2010) defined? He has a few papers from each of those years.